### PR TITLE
Desktop header tweaks

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
This PR tweaks the desktop header for accessibility, adding a "banner" container around the nav bar, and adding a title that shadows any banner background image (for screen readers). This PR does not attempt to convert the background logos to actual `` at this time because of how the configurations are set up, but if this is an issue, let me know.

### CODE REVIEW

**Review by miles-grant-ibigroup — APPROVED** [review]
Can't quite spot all the changes, but I spot no regressions!

File: index.css [hunk 1]
```diff
@@ -162,6 +162,14 @@ input[type="text"]::-ms-clear {
   padding: 1rem 0;
 }
 
+/* Header image or title */
+/* If an icon is used, visually-hide the title (but keep it visible to screen readers. */
```
Comment (AdrianaCeric) [inline index.css:166]: Nice trick!

**Review by AdrianaCeric — APPROVED** [review]
Desktop header lgtm with these new changes. Great attention to accessibility 🥳

**Review by amy-corson-ibigroup — APPROVED** [review]
Thanks Binh, this looks great!

**Pull request merged**